omod/src/main/webapp/resources/scripts/directives/thumbnail.js

```diff
@@ -96,18 +96,18 @@ angular.module('att.widget.thumbnail')
       }
 
       $scope.getPrettyDate = function() {
-        var timeFormat = "DD MMM YY";
+        var dateTimeFormat = ($scope.config.dateFormat).toUpperCase();
         var now = new moment();
         var obsDate = moment($scope.obs.obsDatetime);  // new Date(..) would throw 'Invalid date' on Apple WebKit
         if ( ( obsDate.year() == now.year() ) && ( obsDate.dayOfYear() == now.dayOfYear() ) ) {
-          timeFormat = "HH:mm";
+          dateTimeFormat = "HH:mm";
         }
         else {
           if (obsDate.year() === now.year()) {
-            timeFormat = "DD MMM";
+            dateTimeFormat = dateTimeFormat.replace(/(,?[\.\/\s-])?[yY]+([\.\/\s-])?/ , '');
```

Comment:
Maybe add a little comment here such as
```javascript
// Strips out the year part. Eg 'dd-MMM-yy' → 'dd-MMM', 'YY MM DD' → 'MM DD', ... etc
```

Reply:
Thank you very much @mks-d , I have included the comment.